packages/dataviews/src/utils/filter-sort-and-paginate.ts

```diff
@@ -389,7 +389,10 @@ export default function filterSortAndPaginate< Item >(
 	// Handle sorting.
 	const sortByField = view.sort?.field
 		? _fields.find( ( field ) => {
-				return field.id === view.sort?.field;
+				return (
```

Comment:
This makes sense to me. @oandregal should we add the same check for `groupByField`?

Reply:
I suppose this is fine.

@fushar can you share more in which situations do you run into this? If the field is not sortable, it won't be listed in the view config sort component, and so the user will never trigger a "sorting request" with that field.

Reply:
Just checked the PR description and it's for covering for stored views.